melodic/distribution.yaml

```diff
@@ -6027,6 +6027,12 @@ repositories:
       url: https://github.com/peci1/movie_publisher.git
       version: melodic-devel
     status: developed
+  moving_average:
+    doc:
+      type: git
+      url: https://gitlab.com/InstitutMaupertuis/moving_average.git
+      version: melodic
+    status: maintained
```

Comment:
If you intended to release the package, this is not sufficient, this will only add it to the documentation index.

If you need help getting a package released, follow the [bloom first time release guide](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease)

Reply:
I don't intend to release the package, my goal is just to make it indexed to the documentation